### ISSUE TITLE
Archive page

### ISSUE THREAD

**Issue by EinarBalan** [uclaacm/dev-pathways#43]
More details to be added.

**Comment by reginawang3495** [comment]
@vivianha534 have you worked on this yet (outside of the design)? i was wondering if we could maybe split this up into a few tix and get our react team peeps to work on it?

Editor might run out of easy tix soon

**Comment by doubleiis02** [comment]
blocked by #99
